dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -2,6 +2,7 @@
  * IQueryAssociations object and helper functions
  *
  * Copyright 2002 Jon Griffiths
+ * Copyright 2024 Katayama Hirofumi MZ
```

Comment:
If the changes are minor fixes this is not necessary. The (C) is for when you rewrite large chunks of code in the file.

Reply:
I will delete it.

Reply:
```suggestion
```